spec/index.md

```diff
@@ -445,6 +445,10 @@ Because this uses the lexicographic ordering of princpials, and the byte disting
 
 :::
 
+-   `/subnet/<subnet_id>/nodes/<node_id>/public_key` (blob)
```

Comment:
I recall that we use the singular form when drafting design docs and it is consistent with other paths like /subnet and /canister in singular form.  I prefer the singular form here.

The [MR](https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/13266) of certifying node keys has been merged. If we want to use the plural form, we will have to make a quick fix before it is rolled out. 
cc: @mraszyk @derlerd-dfinity

Reply:
Makes totally sense! Thank you for spotting this and proposing the change. I've updated this spec PR to use the singular form.